bitnami/vault/templates/server/service.yaml

```diff
@@ -9,23 +9,12 @@ kind: Service
 metadata:
   name: {{ include "vault.server.fullname" . }}
   namespace: {{ include "common.names.namespace" . | quote }}
-  labels: {{- include "common.labels.standard" . | nindent 4 }}
+  labels: {{- include "common.labels.standard" ( dict "customLabels" .Values.commonLabels "context" $ ) | nindent 4 }}
     app.kubernetes.io/part-of: vault
     app.kubernetes.io/component: server
-    {{- if .Values.commonLabels }}
-    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
-    {{- end }}
-  {{- if or .Values.server.service.general.annotations .Values.commonAnnotations }}
-  annotations:
-    {{- if .Values.server.service.general.annotations }}
-    {{- include "common.tplvalues.render" ( dict "value" .Values.server.service.general.annotations "context" $) | nindent 4 }}
-    {{- end }}
-    {{- if and .Values.server.metrics.enabled .Values.server.metrics.annotations }}
-    {{- include "common.tplvalues.render" ( dict "value" .Values.server.metrics.annotations "context" $) | nindent 4 }}
-    {{- end }}
-    {{- if .Values.commonAnnotations }}
-    {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
-    {{- end }}
+  {{- if or .Values.server.service.annotations .Values.commonAnnotations }}
```

Comment:
It should be `server.service.general.annotations`

Reply:
fixed at https://github.com/bitnami/charts/pull/18445/commits/ad2828baf71e20c61c08b33cc2ed9534a05c7b9e